### DISCUSSION TITLE
Invert Channel from texture to USD Preview Surface

### DISCUSSION THREAD

**Discussion by Ziethriel** [Autodesk/maya-usd#3234] [Q&A]
Is there a way to invert a channel in a texture (from smoothness to roughness) and keep it after export?

I have an obsessed coworker who refuses to use any software or techniques that cant be easily digested in Unity. So I'm exporting textures from substance painter using the Unity HDRP preset with usd file, however Unity uses smoothness. There appears to be no way to invert the channel so I can use it for our Maya projects, and send it over to Unity without re-making the textures. The connection needs to persist in Maya, but can be broken going into Unity since he re-maps and atlases everything in Unity anyways.

Thanks for your time and help!

For more context, all I need to do is what's in this image, however exporting like this breaks the connection and removes the conversion nodes.

![Invert Channel Image](https://github.com/Autodesk/maya-usd/assets/30094778/a880488c-d560-4904-ad00-8a9a453b7905)

**Comment by sharktacos** [comment]
Maya nodes do not translate well to USD. If you do it instead with Arnold nodes it should work
reverse= aiCompliment
luminance = aiColorToFloat
